java/server/src/org/openqa/selenium/grid/distributor/local/LocalDistributor.java

```diff
@@ -187,7 +186,7 @@ private LocalDistributor add(Node node, NodeStatus status) {
       LOG.fine("Adding host: " + host.asSummary());
       hosts.add(host);
 
-      LOG.info(String.format("Added node %s.", node.getId()));
+      LOG.info(String.format("Added node %s.", node.getId().toUuid()));
```

Comment:
Prefer adding a `toString` to `NodeId`.

Reply:
Thanks! The log displays for example: "Added node org.openqa.selenium.grid.data.NodeId@efb9816f" while using toString() vs "Added node 1915a674-2639-4dc1-8e87-f16b235692fe" while using toUuid(). Do we want that for logs?